src/plugins/intel_cpu/src/nodes/paged_attn.cpp

```diff
@@ -152,18 +152,26 @@ void PagedAttention::execute(dnnl::stream strm) {
         inputs[i] = getSrcMemoryAtPort(i);
     }
 
-    const auto& queryDims = inputs[0]->getStaticDims();
+    auto outDims = inputs[0]->getStaticDims();
+    const auto& keyDims = inputs[1]->getStaticDims();
+    const auto& valueDims = inputs[2]->getStaticDims();
+    // value head_size may be not same with key
+    if (keyDims[1] != valueDims[1]) {
+        // query_ps[1] = num_heads * head_size, key_ps[1] = num_kv_heads * head_size, value_ps[1] = num_kv_heads * v_head_size,
```

Comment:
can remove these comments?

Reply:
The comments are used to explain the following code. The comments are modified to more readable.